pkg/node/node.go

```diff
@@ -51,10 +51,10 @@ func AddNodeLabels(node *v1.Node, labels map[string]string) {
 	}
 }
 
-// RemoveCPUModelNodeLabels removes labels from node with prefix: feature.node.kubernetes.io/cpu-model-*
+// RemoveCPUModelNodeLabels removes labels from node with prefix: feature.node.kubernetes.io/*
 func RemoveCPUModelNodeLabels(node *v1.Node) {
 	for label := range node.Labels {
-		if strings.Contains(label, labelNamespace+"cpu-model-") {
+		if strings.Contains(label, labelNamespace) {
```

Comment:
this assumes our cpu-node-labeller owns the labels of the node (e.g. noone else can add feature.node.kubernetes.io/ labels). Is that right?

Reply:
NFD is using same syntax. So if someone using nfd and this labeller, then labeller deletes all labels.

Reply:
This is not a good idea. We want to be able to use our labeller together with NFD. The idea is to move to NFD once it is clear how our plugins should be integrated, but we should not stomp on NFD's feet.

Can we fix this somehow? NFD uses an annotation where it stores all the labels it set.

Reply:
yes, i was thinking about doing it like NFD.

Reply:
should be fixed now

Reply:
Let's try to add unit/functional tests if feasible.